events/liberty-pass.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Liberty Pass (Battlepass)
+---
+
+## Liberty Pass  
+
+Pe server a fost adaugat un nou sistem de tip battlepass. Acesta consta intr-un set de misiuni saptamanale si 50 tiere pline de premii disponibile pentru a fi deblocate de catre jucatori. Exista atat o varianta free cat si una premium a sistemului. Accesibil folosind tasta `U`.
+
+Liberty Pass se intinde pe o durata de 6 saptamani si ofera undeva la 12-15 quest-uri pe saptamana.  
+
+**Liberty Pass poate fi achizitionat atat pentru contul propriu cat si sub varianta de ticket. Ticketele sunt unice in functie de sezon si pot fi utilizate doar in sezonul in care au fost achizitionate.**  
+**Ticketele dintr-un sezon anterior pot fi convertite in Gold.**  
+
+Deblocarea nivelelor (tiers) are loc pe baza acumularii de experienta. Jucatorii pot obtine experienta la completarea unui quest sau la payday.  
+
+In functie de dificultatea questurilor jucatorii pot primi 8.000, 12.000 sau 16.000 experienta.  
+
+In functie de cat timp  se joaca pe server jucatorii vor primi:  
+- cont standard: 35 experienta/minut jucat  
+- cont Gold: 50 experienta/minut jucat  
+- cont Platinum: 75 experienta pe minut jucat  
+
+Estimativ recompensele primului sezon Liberty Pass sunt:  
+- varianta FREE:   
+    - premii monetare estimate la ~$100.000  
+    - premii estimate la ~7000 Gold  
+- varianta PREMIUM:  
+    - premii monetare estimate la ~$400.000  
+    - premii in Gold estimate la ~$35.000 Gold  
+    - vehicul unic Pantheon Supersport (comparativ cu vehiculele Luxury din categoria $600.000 - $800.000)  
+    - alte premii ale caror valoare nu poate fi estimata  
+	
+## Liberty Pass Boost  
+
+Liberty Pass Boost este un nou tip de boost temporar (12 ore (timp real, nu online pe server) de la timpul activarii ticket-ului (click dreapta, use)) adus conturilor ce poate fi castigat doar in cadrul Liberty Pass.   
+Boost-ul vine sub forma unor tickete ce pot fi activate de catre jucatori oricand.  
+
+Liberty Pass Boost ofera:  
+- +25% mai mult drop la obiecte la joburi  
+- +25% mai multi bani la joburi  
+- +25% mai multi bani la joburile pasive  
+- +25% mai multi bani la rob  
+- dublu streep rep in cadrul roburilor  
+- +100% tier experience la payday-uri  
+
```

Comment:
'Ce ofera Liberty Pass Boost'

/n Liberty Pass Boost ofera:  +25% mai mult drop la obiecte la joburi, +25% mai multi bani la joburi, +25% mai multi bani la joburile pasive, +25% mai multi bani la rob, dublu streep rep in cadrul roburilor, +100% tier experience la payday-uri.